docs/1-getting-started/2-making-a-hero-section/README.md

```diff
@@ -266,9 +267,10 @@ Once pushed to your Jahia instance, if you create a CTA button in your hero sect
 
 You can add as many CTAs as you want to your hero section. This is why we haven't defined our CTA properties on `heroSection` directly: it allows for more flexibility and reusability.
 
-> [!TIP]
-> You can, at all times, take a look at your data tree with the [JCR repository browser](http://localhost:8080/modules/tools/jcrBrowser.jsp).
-> For instance, you should be able to find the content you created at the `sites/<site key>/home/main` path.
-> It can be useful to remove buggy nodes or to understand how your data is persisted.
+:::info
```

Comment:
`:::tip` does not work here?

Reply:
I'm using [bootstrap names](https://getbootstrap.com/docs/5.3/components/alerts/), so we (the academy) supports `success`, `danger`, `warning` and `info`